setup.py

```diff
@@ -10,9 +10,13 @@
     packages=find_packages(),
     python_requires='>=3.6,<4.0',
     install_requires=[
+        'docker>=4.4.0<4.5.0',
+        'mypy==0.812',
```

Comment:
I don't think we use either of these packages.

Reply:
That was for earlier stuff before we were doing the docker swarm.